index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="description" content="Adryan Corcione portfolio">
+</head>
+<body>
+    <h1 style="background-color: #C2ADFF">Adryan Corcione</h1>
+    <section id="about"">
+    <h2>About</h2>
+    <p>I'm a journalist and content marketer learning how to code. I hope to incorporate my love of pop culture into programming projects.</p>
+    </section>
+    <section id="experience"></section>
+    <h2>Experience</h2>
+    <ul>Content Marketing Manager, <i>Rocware</i> (2024-)</ul>
+    <ul>Copywriter, <i>FOLX Health</i> (2022-2023)</ul>
+    <ul>SEO Account Coordinator, <i>NisonCo</i> (2021)</ul>
+    <ul>Journalist, <i>Self-employed</i> (2016-2020)</ul>
+    </section>
+    <section id="social-media">
+    <h2>Connect</h2>
+    <ul><a href="https://www.linkedin.com/in/adryan-corcione/" target="_blank">LinkedIn</a></ul>
+    <ul><a href="https://www.adryancorcione.com/" target="_blank">Website</a></ul>
+    </section>
+    <p><img src="https://images.squarespace-cdn.com/content/v1/656a1fc5ed64a607d3ecf423/7d719ba3-0bde-4065-91d6-40195bd202dc/new+headshot%283%29.png" alt="Adryan Corcione" width="500" height="600"></p>
```

Comment:
It'd be better to put your img tag inside a figure or div parent component than within a p tag.